src/test/lib/storage/table_test.cpp

```diff
@@ -284,4 +285,54 @@ TEST_F(StorageTableTest, StableChunks) {
   EXPECT_EQ((*(*first_chunk)->get_segment(ColumnID{0}))[ChunkOffset{0}], AllTypeVariant{100});
 }
 
+TEST_F(StorageTableTest, LastChunkOfReferenceTable) {
+  const auto reference_table = std::make_shared<Table>(column_definitions, TableType::References);
+  const auto posList = std::make_shared<RowIDPosList>(1);
+  const auto segment_int = std::make_shared<ReferenceSegment>(t, ColumnID{0}, posList);
+  const auto segment_string = std::make_shared<ReferenceSegment>(t, ColumnID{1}, posList);
+  const auto segments = Segments{segment_int, segment_string};
+  const auto second_segment = Segments{segment_int, segment_string};
+  reference_table->append_chunk(segments);
+  reference_table->append_chunk(second_segment);
+  const auto last_chunk = reference_table->get_chunk(ChunkID{1});
+  EXPECT_EQ(reference_table->last_chunk(), last_chunk);
+}
+
+TEST_F(StorageTableTest, CreatePartialHashIndex) {
+  auto hash_index = t->get_table_indexes();
+  EXPECT_TRUE(hash_index.empty());
+  const auto *const world_string = "World";
+  const auto *const hello_string = "Hello";
+  t->append({4, hello_string});
+  t->append({3, world_string});
+  t->append({6, hello_string});
+  t->append({7, "!"});
+  t->append({8, "?"});
+  t->create_partial_hash_index(ColumnID{1}, {ChunkID{0}, ChunkID{1}});
+  hash_index = t->get_table_indexes();
+  ASSERT_EQ(hash_index.size(), 1);
+  const auto created_hash_index = hash_index[0];
+  EXPECT_EQ(created_hash_index->get_indexed_column_id(), ColumnID{1});
+
+  auto access_range_equals_with_iterators_hello = [](auto begin, const auto end) {
+    EXPECT_EQ(std::distance(begin, end), 2);
+
+    EXPECT_EQ(*begin, (RowID{ChunkID{0}, ChunkOffset{0}}));
+    ++begin;
+    EXPECT_EQ(*begin, (RowID{ChunkID{1}, ChunkOffset{0}}));
```

Comment:
Brackets around RowID needed?

Reply:
Yes. Otherwise we get a compilation error: `error: macro "EXPECT_EQ" passed 3 arguments, but takes just 2`

Reply:
This could be fixed by using () instead of {} for the constructor: `EXPECT_EQ(..., RowID(..., ...));`.